### PR TITLE
DP-45 feat: Calendar component

### PR DESCRIPTION
**Pull request by IIshimwe**
# PR Description
Calendar component
# Description of tasks that were expected to be completed
A calendar component that will be used to build trainees & coordinator calendar
# How has this been tested?
```
git clone https://github.com/atlp-rwanda/atlp-pulse-fn.git
cd atlp-pulse-fn
git checkout ft-calendar-component-dp-45
npm install
npm run dev
```
https://devpulse-review-app-27.herokuapp.com/dashboard/calendar

# Screenshots
Screen Shot 2022-07-26 at 12 13 55
Screen Shot 2022-07-26 at 12 14 06

# Please check this Checklist before you submit your PR:
- [x] Calendar in month view should be displayed in a grid
- [x] Calendar is able to move around in different months of the year and even years
- [x] Calendar is able to place an event on it
- [x] Calendar is responsive having mobile and tablet screen size version

# WIP
[Trello card #DP-45](https://trello.com/c/mutc4ERj)

### CODE REVIEW

**Comment by IIshimwe** [comment]
Thanks @feyton I'll take that advantage and try to look at this PR.

**Review by chanceDira — COMMENTED** [review]
Nice job @IIshimwe, 
I think the calendar should be on the dashboard.
Also try to fix the conflit.

File: src/containers/Routes.tsx [hunk 1]
```diff
@@ -16,7 +16,8 @@ function MainRoutes() {
         <Route path="/" element={<Home />} />
         <Route path="/register-organization" element={<OrgRegister />} />
         <Route path="/org-login" element={<Orglogin />} />
-        <Route path="/login-admin" element={<Adminlogin/>}/>
+        <Route path="/login-admin" element={<Adminlogin />} />
+        <Route path="/calendar" element={<Calendar />} />
```
Comment (samyshyaka) [inline src/containers/Routes.tsx:20]: You can move this route to the dashboard routes

**Review by samyshyaka — CHANGES_REQUESTED** [review]

File: src/containers/Routes.tsx (hunk 1 above)
Comment (IIshimwe) [reply src/containers/Routes.tsx:20]: I don't think it is a good idea. This is a component I created a route of it to let you be able to view it but the updated calendar component will not have a route for it. @samyshyaka

**Comment by ericmaniraguha** [comment]
Well done!

**Review by samyshyaka — CHANGES_REQUESTED** [review]
- On the Add event form, the user can only set the start date but there is no end date.
- Also, how will the user specify the time? Currently, they can only select a date.

File: src/Routes.tsx [hunk 2]
```diff
@@ -7,7 +7,12 @@ import Orglogin from './pages/Organization/Orglogin';
 import Pricing from './pages/Pricing';
 import PricingForm from './components/PricingForm';
 import AdminLogin from './pages/Organization/AdminLogin';
+<<<<<<< HEAD
 import Perfomancetraineetable from './components/TraineePerformance';
+=======
+// import Orggooglelogin from './pages/Organization/Orggooglelogin';
+import Calendar from './components/Calendar';
+>>>>>>> Create calendar component
```
Comment (rukundo-kevin) [inline src/Routes.tsx:16]: Resolve these conflicts.

**Review by rukundo-kevin — DISMISSED** [review]

**Review by jeandamascyifuzo — CHANGES_REQUESTED** [review]
Good job @chanceDira  and @IIshimwe 
please fix this issue for small devices
Screen Shot 2022-07-27 at 09 04 02

**Review by samyshyaka — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -43,6 +43,6 @@
   </head>
 
   <body>
-    <div id="tree"></div>
+    <div id="tree" class="hi"></div>
```
Comment (feyton) [inline index.html:46]: Please, remove this class value, it is not serving any purpose anymore

**Review by feyton — APPROVED** [review]
### Approval

@chanceDira this is good work, I left a non critical comment that need to be addressed. Everything else is well implemented

**Review by jeandamascyifuzo — APPROVED** [review]
well done @chanceDira

**Review by ceelogre — CHANGES_REQUESTED** [review]
This should be fixed once for good. There's an open issue with this.
image

**Review by ceelogre — APPROVED** [review]

**Pull request merged**